backend/Dockerfile

```diff
@@ -1,11 +1,23 @@
 FROM ubuntu:18.04
 
 RUN apt-get update && apt-get install -y python3 python3-pip libpq-dev gettext
-WORKDIR /match4everyone-backend
-COPY requirements.txt /match4everyone-backend/requirements.txt
+
+WORKDIR /backend
+COPY requirements.txt /backend/requirements.txt
 RUN pip3 install -r requirements.txt
-COPY requirements.prod.txt /match4everyone-backend/requirements.prod.txt
+COPY requirements.prod.txt /backend/requirements.prod.txt
 RUN pip3 install -r requirements.prod.txt
 
+COPY . .
+
+RUN python3 manage.py makemessages --no-location --ignore 00_old_m4h_matching_code
+RUN python3 manage.py compilemessages
+
+# Change permissions on run/ in case app is later run by non-root user
+# and delete log files as these are created during the above commands when django loads the configuration
+# and will be created non writeable by others than root
+RUN chmod -Rc a+rwX run && rm -v run/*log
```

Comment:
Better solution gosu or su-exec, this should be changed to:
1. ENV variable with uid and gid to be used by container
2. run container as root (remove user param from yml)
3. create user and group inside of container with fixed names (e.g. backend)
4. execute the processes using gosu/su-exec with user from 3